doc/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF

```

Comment:
Just curious: so this file is supposed to be run on M$ Windows?

Reply:
yes